### DISCUSSION TITLE
Suggestions: Loudness equalization & Pause/Resume/Skip audio.

### DISCUSSION THREAD

**Discussion by RyuuzakiL5** [hellcat707hp/GTRadio#15] [Ideas]
Hello,

Thank you for your app, it's being very useful, wonderful. If you don't mind, I would like to suggest some features, but of course, it's just a suggestion:

Loudness equalization it's only needed for custom songs radio, since game's sound files are already at the same sound level. I can use mp3gain to get the songs at the same level as the ads, news, etc. But if by any change you think of that, would be awesome too.

About the other feature, I know that the way it is now, you can pause and resume, but wen you resume or when you lose conduction with Bluetooth or headphones and resune the song is skiped. So this feature could be another button (skip button) and we could have another feature for the pause button, that let us pause and resume from the part of the song or the radio audio that was playing at the time we paused, like a normal player.

**Comment by hellcat707hp** [comment]
Thanks @RyuuzakiL5!

Loudness equalization I hadn't thought of, so definitely something to look into there, just complexity could be notable given how the app uses a wrapper ExoPlayer instance.

Then for the song-skip issue, the "ability"/bug to skip when hitting play/pause is something that has bothered me for a long time and is something I've meant to fix and make time-based. That change might just require a decent refactor of the stations, but maybe not. I'll make an issue for that and link it here!
UPDATE: Here's that issue #20 

Adding a way to skip forward on purpose once that's fixed could be fairly straightforward, so that's another good suggestion.

**Reply by RyuuzakiL5** [reply]
Yeah, don't mind about Loudness equalization, that thing is only needed for custom songs, as the game have audios at the same level (including songs). I just used mp3 gain to adjust that at the same level as news and ads level.
